app/bundles/LeadBundle/Entity/OperatorListTrait.php

```diff
@@ -88,10 +88,20 @@ trait OperatorListTrait
     public function getFilterExpressionFunctions($operator = null)
     {
         $operatorOption = OperatorOptions::getFilterExpressionFunctions();
+        $this->modifyOperators($operatorOption);
 
         return (null === $operator) ? $operatorOption : $operatorOption[$operator];
     }
 
+    /**
+     * Method for modify operators in each model separately.
+     *
+     * @param $operatorOption
+     */
+    public function modifyOperators($operatorOption)
```

Comment:
I'm not sure I'm a fan of this. I know we use inherited trait methods in other places in older code but it's not considered very "clean" by today's standards, especially with a pass by referenced array. I think the cleaner solution for this scenario is to pass an optional array into getFilterExpressionFunctions that gets merged into `$operatorOption`.

Reply:
Or consider refactoring away from this trait.